### ISSUE TITLE
Biceps tendon tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10813]
**Preferred term label:**
Distal biceps tendon tear
Proximal biceps tendon tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
Distal: Ultrasound imaging showing complete discontinuity of the distal biceps tendon or MRI revealing complete distal biceps tendon discontinuity or increased signal intensity in the distal biceps tendon

Proximal: MRI revealing a tear of the proximal aspect of the biceps tendon near its site of origin at the supraglenoid tubercle

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal:
Distal biceps tendon tear

Proximal:
Proximal biceps tendon tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
from the definition, "Rupture" appeared the better primary label.
If this is OK, I will add the two suggested subterms

Adding new term: 
- ID: HP:6001262
- label: Biceps tendon rupture
- parents: [Tendon rupture (HP:0100550)](https://hpo.jax.org/app/browse/term/HP:0100550)
- pmid: PMID:30020607
- def: Discontinuity of the biceps tendon as observed by ultrasound or magnetic resonance (MR) imaging.
- comment: Patients suffering from biceps tendon rupture may complain of sudden sharp pain in the elbow or shoulder, depending on the site of the rupture. Usually, there is a history of sudden eccentric force to a flexed elbow. They may feel an audible pop in the affected arm at the time of the injury. Pain can persist for weeks to months.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10813
- synonyms: Biceps tendon tear(NONE)

**Comment by MickeySegal** [comment]
We will use the subterms, but all should also include the word "tear" somewhere in the various fields.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001367
- label: Distal biceps tendon ruptre
- parents: [Biceps tendon rupture (HP:6001262)](https://hpo.jax.org/app/browse/term/HP:6001262)
- pmid: 
- def: Rupture (tear) of the distal biceps tendon. Ultrasound imaging showing complete discontinuity of the distal biceps tendon or MRI revealing complete distal biceps tendon discontinuity or increased signal intensity in the distal biceps tendon.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10813
- synonyms: Distal biceps tendon tear(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001368
- label: Proximal biceps tendon rupture
- parents: [Biceps tendon rupture (HP:6001262)](https://hpo.jax.org/app/browse/term/HP:6001262)
- pmid: 
- def: Rupture (tear) of the proximal biceps tendon. MRI reveals a tear of the proximal aspect of the biceps tendon near its site of origin at the supraglenoid tubercle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10813
- synonyms: Proximal biceps tendon tear(NONE)
